mbt_gym/gym/TradingEnvironment.py

```diff
@@ -122,10 +119,16 @@ def _get_max_cash(self) -> float:
         return self.max_inventory * self.max_stock_price
 
     def _get_max_depth(self) -> float:
-        return 4.0  # TODO: improve
+        if self.fill_probability_model is not None:
+            return self.fill_probability_model.max_depth
+        else:
+            return 1.0
```

Comment:
I think that it should return None if the environment doesn't have a fill_probability_model.